docs/BreakingChanges/085 Some .NET APIs cause first chance (handled) EntryPointNotFoundExceptions.md

```diff
@@ -0,0 +1,30 @@
+## 85: Some .NET APIs cause first chance (handled) EntryPointNotFoundExceptions
```

Comment:
{
  "Id": "85",
  "Title": "Some .NET APIs cause first chance (handled) EntryPointNotFoundExceptions",
  "Details": "In the .NET Framework 4.5, a small number of .NET methods began throwing first chance EntryPointNotFoundExceptions. These exceptions were handled within the .Net Framework, but could break test automation that did not expect the first chance exceptions. These same APIs break some ApiVerifier scenarios when HighVersionLie is enabled.",
  "Suggestion": "This bug can be avoided by upgrading to .NET Framework 4.5.1. Alternatively, test automation can be updated to not break on first-chance EntryPointNotFoundExceptions.",
  "Link": "https://connect.microsoft.com/VisualStudio/feedback/details/732334/unable-to-find-an-entry-point-named-getcurrentpackageid-in-dll-kernel32-dll",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Diagnostics.Debug.Assert(System.Boolean)",
    "M:System.Diagnostics.Debug.Assert(System.Boolean,System.String)",
    "M:System.Diagnostics.Debug.Assert(System.Boolean,System.String,System.String)",
    "M:System.Diagnostics.Debug.Assert(System.Boolean,System.String,System.String,System.Object[])",
    "M:System.Xml.Serialization.XmlSerializer.#ctor(System.Type)",
    "TBD: Are there more?"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5.1",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Diagnostics.Debug.Assert(System.Boolean)
- [x] M:System.Diagnostics.Debug.Assert(System.Boolean,System.String)
- [x] M:System.Diagnostics.Debug.Assert(System.Boolean,System.String,System.String)
- [x] M:System.Diagnostics.Debug.Assert(System.Boolean,System.String,System.String,System.Object[])
- [x] M:System.Xml.Serialization.XmlSerializer.#ctor(System.Type)
- [ ] TBD: Are there more?